src/style.css

```diff
@@ -1 +1,71 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(../src/fonts/Avenir-Book.ttf), url(../src/fonts/Avenir-Heavy.ttf);
+}
+
+.body {
```

Comment:
The class `.body` is misleading since it suggests styling the `body` element. If you intend to style the `body` tag, you should use the `body` selector without the period. If this is a class, consider renaming it to something more descriptive and less likely to be confused with the `body` element.